### PR TITLE
:alembic: [#33] show integration lib-testapp by  implementing  method…

### LINKED ISSUE #33: Test the integration between library (testapp) and API and Github repo
You can run both the library testapp and the API project with runserver, and connect everthing to see it work.

### LINKED ISSUE #43: As a develop I want to remove duplicated functions
It looks like `sharing_configs/utils.py` has function which do the same as methods in `sharing_configs/client_util.py`:

* `get_folders_from_api` is doing the same as `SharingConfigsClient.get_folder`
* `get_files_in_folder_from_api` is doing the same as `SharingConfigsClient.get_files`

Could you remove them?

### PR DESCRIPTION
**Pull request by frekkenSnork**
Fixes #33 
Fixes #43 
Need feedback: 

1. sharing_configs.admin SharingConfigsExportMixin line 80

- what to do with api data (dict ={"download_url":...,"filename":...}

Thx.

### CODE REVIEW

File: sharing_configs/admin.py [hunk 1]
```diff
@@ -177,7 +178,21 @@ def import_from_view(self, request, **kwargs):
                 filename = form.cleaned_data.get("file_name")
                 obj = SharingConfigsClient()
                 try:
-                    resp_api_dict = obj.import_data(folder, filename)
+                    resp_api_bytes = obj.import_data(folder, filename)
+                    # current filecontent from api: hard-coded: b'example file'
+                    # simulation of api_resp
+                    resp_api_bytes = b"ewogICAgImRhdGVfam9pbmVkIjogIjIwMjItMDMtMDcgMDU6MDQ6MzEiLAogI\
```
Comment (annashamray) [inline sharing_configs/admin.py:184]: We don't want anything hardcoded here

**Comment by annashamray** [comment]
@frekkenSnork Please rebase this branch on master and resolve all conflicts

File: sharing_configs/admin.py (hunk 1 above)
Comment (frekkenSnork) [reply sharing_configs/admin.py:184]: Done

File: .bumpversion.cfg [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [bumpversion]
 commit = False
 tag = False
-current_version = 0.1.0
+current_version = 0.1.1
```
Comment (annashamray) [inline .bumpversion.cfg:4]: Why are release changes here? They are already in the master branch

Comment (frekkenSnork) [reply .bumpversion.cfg:4]: May be as a result of rebase (git pull from master)?

Comment (frekkenSnork) [reply .bumpversion.cfg:4]: Changed: from `current_version = 0.1.1` to `current_version = 0.1.0` in .bumpversion.cfg

Comment (annashamray) [reply .bumpversion.cfg:4]: Please don't change release versions. I think rebase on master with picking only necessary commits should be enough

**Pull request closed without merging**